contracts/token/StandardToken.sol

```diff
@@ -8,7 +8,7 @@ import './ERC20.sol';
 /**
  * @title Standard ERC20 token
  *
- * @dev Implemantation of the basic standart token.
+ * @dev Implemantation of the basic standard token.
```

Comment:
Thanks @jasonbellamy! Would you mind changing `Implemantation` for `Implementation` too?